lib/vtls/wolfssl.c

```diff
@@ -1113,6 +1182,7 @@ const struct Curl_ssl Curl_ssl_wolfssl = {
 #ifdef KEEP_PEER_CERT
   SSLSUPP_PINNEDPUBKEY |
 #endif
+  SSLSUPP_TLS13_CIPHERSUITES |
```

Comment:
If these are curves, we should not set the `SSLSUPP_TLS13_CIPHERSUITES` bit since we don't use the tls13 ciphers for wolfSSL then!

Reply:
So, if I'm going to use TLS 1.3 (and I am because groups are in TLS 1.3) then you have to use a TLS 1.3 cipher suite. So I think this should stay this way. When I say TLS 1.3 cipher suite, I mean one of the following (which are not in TLS 1.2): 

TLS_AES_128_GCM_SHA256
TLS_AES_256_GCM_SHA384
TLS_CHACHA20_POLY1305_SHA256
TLS_AES_128_CCM_SHA256
TLS_AES_128_CCM_8_SHA256

You use these along with groups.

Reply:
Those are also the TLS 1.3 ciphers [recognized by the OpenSSL backend](https://curl.se/docs/ssl-ciphers.html#tls-13-cipher-suites-1).

But since there's no code present in wolfssl.c right now for handling the TLS 1.3 ciphers, there's also nothing that says we're using a wrong name and `curl --tls13-cipher super-cool https://example.com/` will work fine, even if there's no cipher with that name!

Shouldn't we then just remove `SSLSUPP_TLS13_CIPHERSUITES` until we have code that deals with what's being set there?

Reply:
Yes, we should remove it!  I think I misunderstood the purpose of that flag.  I will remove it momentarily.

Reply:
:+1: